src/opts.rs

```diff
@@ -21,3 +21,5 @@ pub const PLUGIN_ALL_OPT: &str = "ALL";
 pub const PLUGIN_OVERRIDE_COMPATIBILITY_CHECK_FLAG: &str = "override-compatibility-check";
 pub const HELP_ARGS_ONLY_TRIGGER_TYPE: &str = "provide-help-args-no-app";
 pub const FROM_REGISTRY_OPT: &str = "REGISTRY_REFERENCE";
+pub const TOKEN: &str = "TOKEN";
+pub const SPIN_LOGIN_TOKEN: &str = "SPIN_LOGIN_TOKEN";
```

Comment:
I'm curious why "login" was chosen as part of the name. In my mind, that seems to imply that this token is to be used for logging in, and other tokens are meant for other operations (e.g. `SPIN_DEPLOY_TOKEN`).

What if we used SPIN_AUTH_TOKEN instead?

Reply:
`SPIN_LOGIN_TOKEN` was me and/or Lann thinking out loud - for me at least there was no particular reason.  I agree `SPIN_AUTH_TOKEN` is better.  Sorry Rajat for the bad steer.

Reply:
no worries. this certainly sounds better. i've changed it to `SPIN_AUTH_TOKEN` now.